drivers/SmartThings/zigbee-smoke-detector/capabilities/selfCheck.yaml

```diff
@@ -0,0 +1,30 @@
+id: stse.selfCheck
+version: 1
+status: proposed
+name: Self Check
+ephemeral: false
+attributes:
+  selfCheckState:
+    schema:
+      type: object
+      properties:
+        value:
+          type: string
+          enum:
+            - idle
+            - selfChecking
+            - selfCheckCompleted
+      additionalProperties: false
+      required:
+        - value
+    setter: startSelfCheck
+commands:
+  startSelfCheck:
+    name: startSelfCheck
+    arguments:
+      - name: state
+        optional: false
+        schema:
+          type: string
+          enum:
+            - startSelfCheck
```

Comment:
this command shouldn't need any arguments

Reply:
Modified.